### DISCUSSION TITLE
What does A> mean in the left margin?

### DISCUSSION THREAD

**Discussion by realh** [prabirshrestha/vim-lsp#1090] [Q&A] [answered]
I've just started using vim-lsp for editing Swift. When I load a Swift file it displays 'A>' in the left margin on a seemingly random line, then the 'A>' moves to the current line when I place the cursor. What does the A> mean? Can I disable it? I like to use the appearance of the left margin as a handy indicator that I've got a problem somewhere in the file.

**Comment by prabirshrestha** [comment]
It means there is a code action. you can use `:LspCodeAction` to see what is there. It is similar to bulb in vscode.

Unfortuantly some language servers always shows it while some lang servers are better at showing.

Yes, you can disable it. https://github.com/prabirshrestha/vim-lsp/pull/1085

**Comment by prabirshrestha** [comment, marked as answer]
This is now documented. Use `:h vim-lsp` and the search docs using  `/A>`.

https://github.com/prabirshrestha/vim-lsp/pull/1099
